LibreHardwareMonitor/Utilities/Logger.cs

```diff
@@ -80,9 +81,18 @@ private void SensorRemoved(ISensor sensor)
             }
         }
 
-        private static string GetFileName(DateTime date)
+        private string GetFileName(DateTime date)
         {
-            return AppDomain.CurrentDomain.BaseDirectory + Path.DirectorySeparatorChar + string.Format(FileNameFormat, date);
+            string pathName = Path.Combine(AppDomain.CurrentDomain.BaseDirectory, LoggingPath);
+            try
+            {
+                if (Directory.Exists(pathName)) Directory.CreateDirectory(pathName);
```

Comment:
Please put this on two lines to match code stile.

Reply:
OK